themes/sas-light-color-theme.json

```diff
@@ -7,7 +7,7 @@
     "keyword": { "foreground": "#3578c5" },
     "sec-keyword": { "foreground": "#224c7c", "bold": true },
     "proc-name": { "foreground": "#224c7c", "bold": true },
-    "comment": { "foreground": "#65802a" },
+    "comment": { "foreground": "#647f29" },
```

Comment:
@jingjiajie, please rebase with latest main branch. Thanks. This line is already on main branch and should not be listed as a change in this PR.